js/index.js

```diff
@@ -40,3 +40,62 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let navItems = document.querySelectorAll('nav > a');
+navItems[0].textContent = siteContent["nav"]["nav-item-1"]
+navItems[1].textContent = siteContent['nav']['nav-item-2'];
+navItems[2].textContent = siteContent['nav']['nav-item-3'];
+navItems[3].textContent = siteContent['nav']['nav-item-4'];
+navItems[4].textContent = siteContent['nav']['nav-item-5'];
+navItems.forEach(x => x.style.color = 'green')
+
+let ctaH = document.querySelector('.cta-text h1');
+ctaH.textContent = siteContent['cta']['h1'];
+
+let image1 = document.querySelector('#cta-img')
+image1.src = "img/header-img.png"
```

Comment:
The image src is provided in the object above. Use that instead! siteContent.nav["img-src"]